docs/source/new-docs/getting-started/installing/index.rst

```diff
@@ -4,3 +4,228 @@
+    export GEOIPS_OUTDIRS=$GEOIPS_PACKAGES_DIR/outdirs
+
+If desired, the GeoIPS environment variables can be added to your
+default shell configuration file (which is typically stored in
+``$HOME/.bashrc``) by running the following commands or something similar:
+
+.. code:: bash
+
+    echo "export GEOIPS_PACKAGES_DIR=$GEOIPS_PACKAGES_DIR" >> ~/.bashrc
+    echo "export GEOIPS_TESTDATA_DIR=$GEOIPS_TESTDATA_DIR" >> ~/.bashrc
+    echo "export GEOIPS_OUTDIRS=$GEOIPS_OUTDIRS" >> ~/.bashrc
+
+Install Dependencies with Conda
+-------------------------------
+
+Conda is a package and virtual environment manager that can
+be used to install the dependencies of GeoIPS.
+
+We recommend using Conda if you are new to GeoIPS, programming or Python.
+
+If you are an expert user, you may want to use the manual
+or Docker installation methods.
+
+Get Conda
+^^^^^^^^^
+
+- Download the appropriate version of `Conda
+  <https://www.anaconda.com/download#downloads>`_,  `Miniconda
+  <https://docs.conda.io/en/latest/miniconda.html>`_, or
+  `Miniforge/Mambaforge <https://github.com/conda-forge/miniforge#download>`_.
+
+  For example, for Linux with Intel chips, one of the following might work:
+
+  .. code:: bash
+
+      wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
+
+  To find your specific version, visit the `Miniforge/Mambaforge github page
+  <https://github.com/conda-forge/miniforge/releases/>`_.
+
+- Make the install script executable and run the installer,
+  following the prompts. Eg.:
+
+  .. code:: bash
+
+      chmod u+x ./Mambaforge-Linux-x86_64.sh
+      # After installing, follow instructions regarding conda init / restarting your terminal !
+      # Make sure you set env vars again if you restart your terminal !
+      ./Mambaforge-Linux-x86_64.sh
+
+  .. code:: bash
+
+      rm -f ./Mambaforge-Linux-x86_64.sh
+
+Create and activate a conda environment with dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Next we'll create a conda environment named ``geoips`` that contains all system
+requirements for GeoIPS.
+
+.. code:: bash
+
+    mamba create -y -n geoips -c conda-forge python=3.10 openblas git
+
+.. note::
+
+    If you would like to run plugins that require fortran, you will need to install ``gfortran``.
+
+.. warning::
+
+        If you are using an ARM machine, you may need to install ``pyhdf`` with conda.
+        If you run into ``fatal error: 'hdf.h' file not found`` when installing ``pyhdf``,
+        you may need to install ``pyhdf`` with conda. You can do this with the following command:
+
+        .. code:: bash
+
+            mamba create -y -n geoips -c conda-forge python=3.10 openblas git **pyhdf**
+
+Activate the conda environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You will need to run ``mamba activate geoips`` (or ``conda activate geoips``)
+every time you want to run or work on GeoIPS. Eg.
+
+.. code:: bash
+
+    mamba activate geoips
```

Comment:
I think conda activate geoips all the time - mamba activate doesn't seem to actually work.

Reply:
Good point - changed